src/workflows/airqo_etl_utils/bigquery_api.py

```
@@ -1400,6 +1400,55 @@ def fetch_raw_site_data_for_forecast_jobs(
+            "site_hourly_measurements_for_forecast_jobs"
+        ).format(
+            consolidated_table=f"`{self.consolidated_data_table}`",
+            sites_table=f"`{self.sites_table}`",
+            start_timestamp=start_timestamp.strftime("%Y-%m-%d %H:%M:%S"),
+            end_timestamp=end_timestamp.strftime("%Y-%m-%d %H:%M:%S"),
+            min_hours=min_hours,
+        )
+
+        try:
+            return self.execute_data_query(query=query)
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)